app/assets/stylesheets/application.css.scss

```diff
@@ -15,8 +15,29 @@
  *= require foundation_and_overrides
  */
 
+.logo {
+  background-color: #d9b82d;
+  border-color: #111111;
+  margin-bottom: 0px;
+  margin-top: 10%;
+  text-align: center;
+  text-shadow: -1px 0 white, 0 1px white, 1px 0 white, 0 -1px white;
+}
+
+.logo h1 {
+  border-radius: 0.1em;
+  color: #111111;
```

Comment:
Color literals like `#111111` should only be used in variable declarations; they should be referred to via variable everywhere else.